index.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```

Comment:
what are these used for?

Reply:
I had removed these lines before commiting. i am removing them.

Reply:
please check. these lines are not present in my code.

Reply:
ok then